packages/api-queries/src/site-scan.ts

```diff
@@ -55,3 +63,12 @@ export const fixThreatMutation = ( siteId: number ) =>
 			queryClient.invalidateQueries( siteScanHistoryQuery( siteId ) );
 		},
 	} );
+
+export const fixThreatsMutation = ( siteId: number ) =>
+	mutationOptions( {
+		mutationFn: ( threatIds: number[] ) => fixThreats( siteId, threatIds ),
+		onSuccess: () => {
+			queryClient.invalidateQueries( siteScanQuery( siteId ) );
+			queryClient.invalidateQueries( siteScanHistoryQuery( siteId ) );
```

Comment:
Since this API initiates an async process to fix threats, I wouldn’t invalidate those queries yet because we don’t expect any updates to active threats (or history) immediately after the API resolves.